### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**
### Reason for change

[Create call to Get subscriptions](https://github.com/alma/alma-php-client/pull/75)
[Add code coverage github action](https://github.com/alma/alma-php-client/pull/73)

### Checklist for authors and reviewers

- [ ] The title of the PR uses business wording, not technical jargon, for the changelog readers to understand it
- [ ] The PR implements the changes asked in the referenced task / issue
- [ ] The automated tests are compliant with the [testing strategy](https://www.notion.so/almapay/Backend-testing-strategy-06c642cec1bf47b9b8feca3a91ea8d4a)
- [ ] The tests are relevant, and cover the corner/error cases, not only the happy path
- [ ] You understand the impact of this PR on existing code/features
- [ ] The changes include adequate logging and Datadog traces
- [ ] Documentation is updated (API, developer documentation, ADR, Notion...)

### CODE REVIEW

File: .github/workflows/code-coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+jobs:
+  ci:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: ./
+
+    steps:
+      - uses: actions/checkout@v4
+
```
[the remark this answers is not in the document]
Comment (joyet-simon) [reply .github/workflows/code-coverage.yaml:19]: False Positive

File: .github/workflows/code-coverage.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: PHPUnit Coverage Report
```
[the remark this answers is not in the document]
Comment (joyet-simon) [reply .github/workflows/code-coverage.yaml:1]: False Positive

[the remark this answers is not in the document]
Comment (joyet-simon) [reply .github/workflows/code-coverage.yaml:1]: False Positive

[the remark this answers is not in the document]
Comment (joyet-simon) [reply .github/workflows/code-coverage.yaml:1]: False Positive

**Review by Francois-Gomis — APPROVED** [review]

**Pull request merged**